dashboard/src/app/stacks/stack-details/stack.controller.ts

```diff
@@ -197,15 +197,9 @@ export class StackController {
       this.loading = false;
       this.prepareStackData();
     }, (error: any) => {
-      if (error && error.status === 304) {
-        this.loading = false;
-        this.stack = this.cheStack.getStackById(this.stackId);
-        this.prepareStackData();
-      } else {
-        this.$log.error(error);
-        this.loading = false;
-        this.invalidStack = error.statusText + error.status;
-      }
+      this.$log.error(error);
+      this.loading = false;
+      this.invalidStack = error.statusText + error.status;
```

Comment:
do we need some extra separator between statusText and error status like a space or colon ?

Reply:
done